pennylane_sf/tf.py

```diff
@@ -0,0 +1,275 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.

```

Comment:
Nope, as TF is not a strict dependency of this device. Forcing this device to be imported on plugin import would cause issues for users that don't have TF installed.